Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1084,6 +1084,19 @@ CSSRelativeColorSyntaxEnabled:
     WebCore:
       default: true
 
+CSSRhythmicSizingEnabled:
+  type: bool
+  status: unstable
```

Comment:
Might be worth making it `status: testable` so it is enabled in WPT.

Reply:
Ah, yes that is a good idea :)

Done!